### DISCUSSION TITLE
Regarding the downstream analysis question

### DISCUSSION THREAD

**Discussion by FP-10** [GreenleafLab/ArchR#790] [Questions / Documentation]
Hi

I have two samples, the control group and the experimental group.
After clustering, the cells are well integrated. In the downstream analysis, how can I distinguish the source of cells and analyze them.
For example, in a cluster, how can I compare the changes in the accessibility of differences between the control group and the experimental group.

**Comment by rcorces** [comment]
See `cellColData`
